pom.xml

```diff
@@ -198,40 +198,23 @@
             </dependency>
 
             <!-- For sl4j/logback logging, see https://wiki.jasig.org/display/PLT/Logging+Best+Practices -->
```

Comment:
Probably a good time to update this outdated link.  New one is:  https://apereo.atlassian.net/wiki/spaces/PLT/pages/101548207/Logging+Best+Practices

Reply:
These logging deps are being moved into the parent pom, and I've set up a ticket to enhance/revise the logging docs - https://github.com/uPortal-Project/uPortal-website/issues/14